libs/pasearchalgolib/SearchAlgoConfigurationFileReader.cpp

```diff
@@ -172,7 +172,6 @@ namespace mkc_searchalgo
       std::cout << "First date random access: " << series->getDateValue(series->beginRandomAccess(),0) << std::endl;
       std::cout << "First date sorted access: " << series->getFirstDate() << std::endl;
       series->syncronizeMapAndArray();
-      syntheticTimeSeriesCreator->rewriteTimeFrameFile(timeFrameIdToLoad, series);
```

Comment:
Is this a left over artifact?

Reply:
I kept in the "mixing" that was always done but needed to remove the file rewrite since this was causing issues with the multi-threading (one thread was reading while another was writing).